src/language-css/utils.js

```diff
@@ -461,7 +461,17 @@ function isAtWordPlaceholderNode(node) {
   );
 }
 
+function locStart(node) {
+  return node.source.startOffset;
```

Comment:
Why not put these to `language-css/loc.js`?

Reply:
I was worried about Circular dependency in future.

Reply:
In `language-js`, `isBlockComment` function was in `comment.js`,when we  start use functions from `utils.js` in `comment.js`, I have to move it to `utils.js`.

Reply:
On the other hand, files like `utils`, `helpers`, etc. are an antipattern. Usually it's better to move functions to other files with more meaningful names.